frontend/src/APIcall.js

```diff
@@ -0,0 +1,14 @@
+// fetching the GET route from the Express server which matches the GET route from server.js
+const getData = async () => {
+  try {
+    const response = await fetch('/express_backend');
+    const body = await response.json();
+    if (response.status !== 200) {
+      throw Error(body.message);
+    }
+    return body;
+  } catch (e) {
+    throw e;
+  }
+};
```

Comment:
the` /exress_backend` is an example of an API endpoint, but in this project we do not need an "express_backend" endpoint, we need things like `/users`, `/login`, `/registered`. 
also,  what is the purpose of this function? is this supposed to be the function that will be used for all the API calls in the project? is it supposed to be a specific call to a specific endpoint?

Reply:
Yes, I understand that `/express_backend` a general example, but should I change it ? because I meant to write a general example.
The purpose of this function is to send a request from the client to server  (an API call) and it well not be used for all the API calls, there is another calls like POST method

Reply:
So you don't mean this API call to be actually used? this is just an example of how to create an API call from the client and how to create an API endpoint in the server? 
Instead of just an example, turn this into a call that can be used in your project, I gave you a few endpoint examples in the previous comment.